lib/peep.rb

```diff
@@ -0,0 +1,32 @@
+require 'timecop'
+class Peep
+  attr_reader :id, :texts, :time
+
+  def initialize(id, texts, time)
+    @id = id
+    @texts = texts
+    @time = time
+  end
+
+  def self.all
+    result = DatabaseConnection.query("SELECT * FROM peeps;")
+    result.map do |row| peep = Peep.new(row["id"], row["texts"], row["time"]) end
+  end
+
+  def self.create(texts)
+    DatabaseConnection.query("INSERT INTO peeps (texts) VALUES ('#{texts}');")
+    # Peep.new(result["id"], result["texts"])
```

Comment:
After adding the texts to your db, how do you show user what they have just posted?